source/libraries/desktop/Butler/App/SocialLogin.hs

```diff
@@ -42,14 +74,18 @@ socialLoginApp =
 
 startSocialLoginApp :: AppContext -> ProcessIO ()
 startSocialLoginApp ctx = do
+    let appState = APPState "My secret text"
+        memAddr = "social-login-" <> showT ctx.wid <> ".bin"
+    (_, state) <- newProcessMemory (from memAddr) (pure appState)
```

Comment:
Similar to the todo manager, we could have a single list of secret for the workspace, e.g.:

```suggestion
    (_, state) <- getSharedDynamic ctx.shared.dynamics "social-secret" do
        newProcessMemory (from memAddr) (pure appState)
```

Reply:
Thanks for the review. I've pushed a new revision to use getSharedDynamic.